packages/core/src/external/carequality/ihe-gateway-v2/constants.ts

```diff
@@ -16,4 +16,4 @@ export const namespaces = {
   xsiType: "xsd:string",
 };
 
-export const expiresIn = 5;
+export const expiresIn = 20;
```

Comment:
need saml timestamp to last longer!
5 mins too short if we are doing retries with the same request